common/src/main/java/com/wynntils/gui/render/MapRenderer.java

```diff
@@ -142,4 +142,14 @@ public static float getRenderZ(Poi poi, float mapCenterZ, float centerZ, float c
         double distanceZ = poi.getLocation().getZ() - mapCenterZ;
         return (float) (centerZ + distanceZ * currentZoom);
     }
+
+    public static float getRenderX(float gameX, float mapCenterX, float centerX, float currentZoom) {
+        double distanceX = gameX - mapCenterX;
+        return (float) (centerX + distanceX * currentZoom);
+    }
+
+    public static float getRenderZ(float gameZ, float mapCenterZ, float centerZ, float currentZoom) {
+        double distanceZ = gameZ - mapCenterZ;
+        return (float) (centerZ + distanceZ * currentZoom);
+    }
```

Comment:
This is identical to `getRenderX` and `getRenderZ` but not poi, make one of the methods call the other?

Reply:
I don't think this is even used now